### ISSUE TITLE
Can't compile on Windows

### ISSUE THREAD

**Issue by eteeselink** [appsignal/appsignal-elixir#477]
When trying to compile 1.10.4 on Windows, i get this error:

![image](https://user-images.githubusercontent.com/703546/55803510-beaeae00-5ada-11e9-8c26-f9f194e8cf4b.png)

Now, I don't need AppSignal to _work_ on Windows, but I'd like to be able to dev on an appsignal-enabled codebase on windows.

If anyone else is running into this, my workaround was to copy `id.exe` out from the `Git\usr\bin` directory into the PATH.

**Comment by jeffkreeftmeijer** [comment]
Hey @eteeselink,

Thanks for reporting this issue. We should disable AppSignal completely on Windows, but it looks like we initialize the installation report before checking the architecture here, which checks `id -u` on the host machine. We're working on a patch that will make sure it doesn't break compilation. Could you try starting your app locally using the `report-arch` branch for the appsignal dependency, to see if that gets you a step further?

```
{:appsignal, github: "appsignal/appsignal-elixir", branch: "report-arch"}
```

**Comment by tombruijn** [comment]
Fixed in #479. Will release next week in a patch release.
